### DISCUSSION TITLE
Number of items closed for discussions

### DISCUSSION THREAD

**Discussion by Piedone** [github-community-projects/issue-metrics#300] [Q&A] [answered]
On May 1, the action generated discussion metrics for the Orchard Core project, see [here](https://github.com/OrchardCMS/OrchardCore/issues/15929#:~:text=notready%20%2Dis%3Adraft-,Discussion%20Metrics,-Metric). (This was with `@v3` so I suppose it used v3.3.0.) As you can see, it indicates 4 discussions closed. We had 10 answered discussions in that period.

The metrics generated on 1 June, see [here](https://github.com/OrchardCMS/OrchardCore/issues/16210#:~:text=notready%20%2Dis%3Adraft-,Discussion%20Metrics,-Metric) indicate 0 discussions closed while 9 were answered.

What's "closed" supposed to mean for discussions? I suppose it makes sense, since we had more than 0, but I guess it's not indicating answered discussions. BTW [we generate](https://github.com/OrchardCMS/OrchardCore/blob/main/.github/workflows/community_metrics.yml#L56) these metrics with `HIDE_TIME_TO_CLOSE: true`.

Thank you!

**Comment by zkoppert** [comment]
👋🏻 Hi @Piedone! [Closed](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion) is a separate state from [answered](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer) with regards to discussions. As an admin on a repo, I see an option to close the discussion which can happen independently from marking a response as an answer.
<img width="925" alt="Screenshot 2024-06-07 at 3 53 24 PM" src="https://github.com/github/issue-metrics/assets/6935431/6a177362-0297-4bcb-9dc5-aab0e3588587">

As to why one would close a discussion vs just mark the answer and leave it open, I'm not actually sure what that is supposed to signal in terms of meaning. 🤷🏻

**Reply by Piedone** [reply]
I see, thanks. I don't think "closed" makes much sense for discussions, then. Can we hide it from the metrics if 'HIDE_TIME_TO_CLOSE: true' doesn't hide it?

**Reply by zkoppert** [reply, marked as answer]
That should be hiding it from the summary section as well as the detailed table. I've opened an [issue](https://github.com/github/issue-metrics/issues/308) to track that bug and its fix.

**Reply by Piedone** [reply]
Thank you!
